src/test/java/racingcar/TestUtils.java

```diff
@@ -0,0 +1,21 @@
+package racingcar;
+
+import java.util.List;
+import racingcar.domain.Car;
+import racingcar.domain.movable.RandomMovableStrategy;
+
+public class TestUtils {
+
+    private TestUtils() {}
+
+    static void carMove(Car car, List<Boolean> moves) {
+        for (boolean carMoved: moves) {
+            car.run(new RandomMovableStrategy() {
```

Comment:
[람다](https://khj93.tistory.com/entry/JAVA-%EB%9E%8C%EB%8B%A4%EC%8B%9DRambda%EB%9E%80-%EB%AC%B4%EC%97%87%EC%9D%B4%EA%B3%A0-%EC%82%AC%EC%9A%A9%EB%B2%95)를 사용하면 아래와 같이 작성할 수 있어요!

참고만해주세용~

```suggestion
            car.run(() -> carMoved);
```